cmake/msvc/config.h

```diff
@@ -32,8 +32,9 @@ static inline float rintf(float x){return (x > 0.0f)? floorf(x + 0.5f) : ceilf(x
 ////////////////////////////////////////////////////////////////////////
 // math constants
 ////////////////////////////////////////////////////////////////////////
-#define INFINITY HUGE_VAL
-
+#if (!_MSC_VER >= 1900)
+# define INFINITY   HUGE_VAL
```

Comment:
Shouldn't that be `+#if (_MSC_VER < 1800)` ? Since in my `<math.h>` from MS Visual Studio (`_MSC_VER=1800`), I do have:
```
#define INFINITY   ((float)(_HUGE_ENUF * _HUGE_ENUF))
```